tests/foreign_object/models/article.py

```diff
@@ -87,7 +87,7 @@ class ArticleTranslation(models.Model):
 
     class Meta:
         unique_together = ('article', 'lang')
-        ordering = ('active_translation__title',)
+        # ordering = ('active_translation__title',)
```

Comment:
@timgraham I had to comment this line. because it broke the tests. any idea?

Reply:
Seems fine to remove. `ArticleTranslation.objects.all()` fails with `FieldError: Cannot resolve keyword 'active_translation' into field.`. It looks like that was always broken.

Reply:
@timgraham I removed the line